runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -432,6 +433,11 @@ exec::IExecutor *ExecutorFactory::createLinearExecutor(
 
   auto code_map = builder.releaseCodeMap();
 
+  auto mmr = options.minmax_filepath.empty()
+               ? nullptr
+               : std::make_unique<exec::MinMaxRecorder>(options.minmax_filepath,
+                                                        lowered_graph->graph(), backend_contexts);
```

Comment:
@hseok-oh

- It saves reference to `backend_contexts`, which is going to `std::move`'d.
  - It would be better to borrow `backend_contexts` from `LinearExecutor` after `std::move`.
- It assumes `LinearExecutor` outlives `MinMaxRecorder`. 

I will add a commit on top of previous code.

Reply:
As I guessed, previous code gets segmentation fault because it accesses to std::move'd objects. I added fix commit.